cmd/system-probe/config/config.go

```diff
@@ -176,6 +177,15 @@ func load() (*Config, error) {
 		npmEnabled = true
 	}
 
+	deprecatedEnableHttpMonitoringKey := key(netNS, "enable_http_monitoring")
+	enableHttpMonitoringKey := key(smNS, "enable_http_monitoring")
+
+	if cfg.IsSet(deprecatedEnableHttpMonitoringKey) {
+		log.Infof("%q is deprecated, use %q instead",
+			deprecatedEnableHttpMonitoringKey, enableHttpMonitoringKey)
+		cfg.Set(enableHttpMonitoringKey, cfg.GetBool(deprecatedEnableHttpMonitoringKey))
+	}
```

Comment:
Could these changes be in `pkg/network/config/config.go` instead? I'm trying to keep this file as lean as possible, and only having changes related to enabling/disabling modules and system-probe wide settings. 

(I realize there is some existing code that should be moved too).

Reply:
Done